src/components/Learningpath/Learningpath.jsx

```diff
@@ -75,6 +76,34 @@ const Learningpath = ({
     lastUpdatedDate.getMonth() < 10 ? '0' : ''
   }${lastUpdatedDate.getMonth()}.${lastUpdatedDate.getFullYear()}`;
 
+  const { contentTypes } = constants;
+
+  const mappedLearningsteps = learningsteps.map(step => {
+    let type = '';
+    switch (step.type) {
+      case 'INTRODUCTION':
+        type = contentTypes.LEARNING_PATH;
+        break;
+      case 'TEXT':
+        type = contentTypes.SUBJECT_MATERIAL;
+        break;
+      case 'TASK':
+      case 'QUIZ':
+        type = contentTypes.TASKS_AND_ACTIVITIES;
+        break;
+      case 'MULTIMEDIA':
+        type = contentTypes.ASSESSMENT_RESOURCES;
+        break;
+      default:
+        type = contentTypes.LEARNING_PATH;
+        break;
+    }
+    return {
+      ...step,
+      type: type,
+    };
+  });
+
```

Comment:
Ser at du har gjort det helt likt som i eksempelet i designmanualen. I designmanualen hentes data direkte fra læresti-apiet, og det har ikke ressurs-typer på artikkelen. Derfor måtte det gjøres en slik mapping. 
I front-end her så har dere vel tilgang til ressurs-type på artikkelen og kan bruke den direkte?

Reply:
Så vidt jeg kan se får den bare den samme ressurstypen i caps som fra læresti-apiet. Men kanskje det da hadde vært mer hensiktsmessig å enten mappe direkte i LearningpathMenu, eller eventuelt bruke den typenotasjonen som ligger i apiet?
<img width="394" alt="Skjermbilde 2021-02-24 kl  09 17 44" src="https://user-images.githubusercontent.com/55005434/108968515-28a9af80-7681-11eb-8d71-21263238be29.png">

Reply:
På introduksjon er det sikkert ikke ressurser. Men se på f.eks steg to: @gunnarvelle nevnte noe om dette

![image](https://user-images.githubusercontent.com/1061895/108971640-cd78bc80-7682-11eb-903b-13618f27e1c4.png)

Reply:
aha, der nede ja! Da skal jeg bruke det i stedet.

Reply:
De ressursene har ikke helt de samme typene som din mapping i designmanualen. For eksempel denne har typen MULTIMEDIA fra læringssti-api, men task som resource. Så den får ikonet task når jeg mapper med resourcetypes. Men den inneholder både en film og en oppgave, så det går kanskje greit?
<img width="366" alt="Skjermbilde 2021-02-24 kl  10 29 21" src="https://user-images.githubusercontent.com/55005434/108979897-690e2b00-768b-11eb-96d5-827fd8b8500b.png">

Reply:
Det er helt flott :)